### PR TITLE
✨(core) upgrade acme to v2 API

### PR DESCRIPTION
**Pull request by jmaupetit**
## Purpose

OpenShift-acme project recently added the Let's encrypt API v2 support. Switching to this version of the API is required since previous endpoint were considered as deprecated and closed.

## Proposal

- [x] integrate latest `openshift-acme` objects [1]

[1] https://github.com/tnozicka/openshift-acme/tree/master/deploy/single-namespace

### CODE REVIEW

File: core_apps/acme/templates/services/app/cm_live.yml.j2 [hunk 1]
```diff
@@ -0,0 +1,11 @@
+kind: ConfigMap
```
Comment (lunika) [inline core_apps/acme/templates/services/app/cm_live.yml.j2:1]: I don't understand how these config maps are used ?

Comment (madmatah) [reply core_apps/acme/templates/services/app/cm_live.yml.j2:1]: If I understand correctly... the `openshift-acme` controller get these ConfigMap by matching their labels :

 - https://github.com/tnozicka/openshift-acme/blob/8b54945f9457b1b960fea133bce359679eee5d62/pkg/api/internal.go#L22-L25
- https://github.com/tnozicka/openshift-acme/blob/8b54945f9457b1b960fea133bce359679eee5d62/pkg/controllerutils/controllerutils.go#L35

And they define the available certificate issuers.

**Review by madmatah — COMMENTED** [review]
In the [single-namespace deployment example](https://github.com/tnozicka/openshift-acme/tree/master/deploy/single-namespace), a [service account](https://github.com/tnozicka/openshift-acme/blob/master/deploy/single-namespace/serviceaccount.yaml) is defined. Why isn't it required here ?

**Comment by madmatah** [comment]
> In the [single-namespace deployment example](https://github.com/tnozicka/openshift-acme/tree/master/deploy/single-namespace), a [service account](https://github.com/tnozicka/openshift-acme/blob/master/deploy/single-namespace/serviceaccount.yaml) is defined. Why isn't it required here ?

Oh sorry I did not see the `sa.yml` file... :man_facepalming:

**Review by madmatah — APPROVED** [review]

File: core_apps/acme/templates/services/app/cm_live.yml.j2 (hunk 1 above)
Comment (jmaupetit) [reply core_apps/acme/templates/services/app/cm_live.yml.j2:1]: You'll have to checkout the `openshift-acme` code to understand this part.

Comment (jmaupetit) [reply core_apps/acme/templates/services/app/cm_live.yml.j2:1]: (I didn't see @madmatah comment before answering)

**Pull request merged**